### DISCUSSION TITLE
Underscores in indexes are splitting words

### DISCUSSION THREAD

**Discussion by ItsRobbAllen** [ravendb/ravendb#18429] [Q&A]
We have an index to look up email addresses, but if the email address is john_doe@mydomain.com, we cannot search for `john_doe` because the index breaks it down to `john` and `doe`.

How do we index something with an underscore & keep the underscore?

**Comment by krivulcik** [comment]
You might want to use a different analyzer (see the explanation of the default analyzers here: https://ravendb.net/docs/article-page/6.0/Csharp/indexes/using-analyzers ).
